<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmotionClass</class>
 <widget class="QMainWindow" name="EmotionClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FEAtuREs</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Real Time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>290</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Registra</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button3">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>290</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Gill Sans MT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Carica e Analizza</string>
    </property>
   </widget>
   <widget class="QLabel" name="imm">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>531</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Webcam Interna</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Webcam Esterna</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="emotion.qrc"/>
 </resources>
 <connections/>
</ui>
